styles/components/_bankBoard.scss

```diff
@@ -1,28 +1,29 @@
 .bank {
   &__board {
-    display: inline-flex;
-    width: 90vw;
-    align-content: space-around;
+    display: flex;
+    flex-wrap: wrap;
     align-items: center;
-    justify-content: space-around;
-    background-color: wheat;
+    justify-content: space-between;
+    background-color: $color-primary;
+    border-radius: $border-radius;
 
     &__container {
```

Comment:
The name suggests it is a bank board container, perhaps bank__item? bank__animal? bank__animal-container?